fintrack/banks/admin.py

```diff
@@ -1,6 +1,16 @@
 from django.contrib import admin
 
-from . models import Bank, Branch
+from .models import Bank, Branch
 
-admin.site.register(Bank)
-admin.site.register(Branch)
+
+@admin.register(Bank)
+class BankAdmin(admin.ModelAdmin):
+    list_display = ('id', 'name')
+    search_fields = ('name',)
+
+
+@admin.register(Branch)
+class BranchAdmin(admin.ModelAdmin):
+    list_display = ('id', 'name', 'code', 'address', 'phone_number', 'bank')
+    list_filter = ('bank',)
```

Comment:
it's not recommended to include foreign keys into filter
reason is, when there is large amount of different objects in foreign key listing, it breaks the page